### ISSUE TITLE
[docs]:  duplicate id on page when css-custom-properties is used multiple times

### ISSUE THREAD

**Issue by zeroedin** [RedHat-UX/red-hat-design-system#1099]
This is an [upstream in PFE](https://patternflyelements.com/components/tabs/) as well but tagging it here for visibility.

The issue is with `{% renderCssProperties %}` being used multiple times on the code tab for components with sub components.

Can see this issue in this [Deploy Preview](https://deploy-preview-1098--red-hat-design-system.netlify.app/elements/audio-player/code/)

`#styling-hooks` gets rendered in each section. 

@bennypowers my assumption is modifying that `id` would be considered a breaking change to downstream repos.  Should we add an additional method be added that would allow for customization of that ID to get a change out faster?

**Comment by zeroedin** [comment]
PR: https://github.com/patternfly/patternfly-elements/pull/2527

**Comment by bennypowers** [comment]
@zeroedin once that pr releases (preferably after timestamp controller lands) we can update here and close this issue

**Comment by zeroedin** [comment]
> @zeroedin once that pr releases (preferably after timestamp controller lands) we can update here and close this issue

Yes sir, as soon as that gets released we can bump deps here and ill close this with that PR.
